playbooks/de-deploy-service-nginx.yaml

```diff
@@ -25,7 +25,7 @@
   roles:
     - role: de-deploy-service
       has_configs: false
-      has_iplant_data: false
+      has_data_container: false
```

Comment:
I renamed this simply because the data container name isn't always `iplant_data` now.